src/collections/members/randy-chilau/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+name: Randy Chilau
```

Comment:
Hi @kotkaravishkar,

Thanks 🙏  for working on this.

One small correction that I missed, please use the name "Randy Lau"

Otherwise it looks good, thanks again!

Cheers,
Randy

Reply:
Updated.